src/main/java/io/appform/dropwizard/actors/actor/BaseActor.java

```diff
@@ -100,7 +100,8 @@ protected BaseActor(
     }
 
 
-    abstract protected boolean handle(Message message) throws Exception;
+    abstract protected boolean handle(Message message,
```

Comment:
There should be a delegate to the older function here. Otherwise this will completely break all older code. It also makes sense to have people implement the simpler function in general and move to this if needed.

Reply:
keeping both the handle functions older and new one will not be possible now since the MessageHandlingFunction has changed now to (T t, X x), clients using new version will have to override the newer handle function. Clients using bundle don't have option to choose the handle function impl, it gets decided in the bundle itself.

Reply:
Check the modifications suggested to MessageHandlingFunction

Reply:
done.